composer.json

```diff
@@ -12,15 +12,15 @@
   ],
   "minimum-stability": "stable",
   "require": {
-    "php": "~8.1.0 || ~8.2.0 || ~8.3.0",
-    "behat/behat": "^3.11",
+    "php": "~8.2.0 || ~8.3.0 || ~8.4.0",
+    "behat/behat": "^3.15",
     "ext-json": "*",
-    "symfony/http-client": "^5.4 || ^6.3",
+    "symfony/http-client": "^7.2",
```

Comment:
Are we definitely safe to drop support for symfony 6.x? Probably should mention in the changelog.

Reply:
6.4.6 is the lowest we can go